sdk/tests/conformance2/reading/read-pixels-pack-parameters.html

```diff
@@ -41,6 +41,7 @@
 "use strict"
 
 var wtu = WebGLTestUtils;
+var clearColor = [1, 2, 3, 4];
```

Comment:
Since this isn't passed to the OpenGL function clear() (as far as I can tell), naming this variable differently, like "initialColor", would make the code more clear.